docs/sources/flow/getting-started/distribute-prometheus-scrape-load.md

```diff
@@ -18,23 +18,17 @@ auto-distribution allows a fleet of agents to work together to dynamically
 distribute their scrape load, providing high-availability.
 
 {{% admonition type="note" %}}
-Clustering is a [beta]({{< relref "../../stability.md#beta" >}}) feature. Beta features are subject to breaking
+Clustering is a [beta](https://www.grafana.com/docs/agent/<AGENT_VERSION>/stability#beta) feature. Beta features are subject to breaking
```

Comment:
This link does work but looks like it should be a relative link like the rest are I think.

Reply:
We can't use doc refs within an admonition/note. It's a limitation of the Hugo side of things.  I have to made a decision... fixed link like this, completely remove the link, or convert these Notes from admonitions to block quotes. I'm leaning towards block quotes because the rendered result is the same as an admonition and I can use the doc/reference tooling to make sure that the readers stay within the doc tree (OSS or Cloud).